src/main/java/se/sundsvall/casedata/api/model/enums/AttachmentCategory.java

```diff
@@ -275,7 +269,21 @@ public enum AttachmentCategory {
 	///////////////////////////////////
 	ERRAND_SCANNED_APPLICATION("Ärende (Skannad ansökan)", null),
 	SERVICE_RECEIPT("Delgivningskvitto", null),
-	OTHER_ATTACHMENT("Övriga bilagor", null);
+	OTHER_ATTACHMENT("Övriga bilagor", null),
+
+	///////////////////////////////////
+	// MEX
+	///////////////////////////////////
+	LEASE_REQUEST("Förfrågan arrende", null),
+	RECEIVED_MAP("Karta inkommen", null),
+	RECIVIED_CONTRACT("Avtal inkommit", null),
```

Comment:
Different spelling of "received", same in the story but should probably be fixed.

Reply:
Good catch. Fixed that and also changed translation on "F-skattsedel"-enum.